feature/experimental/gribi/ate_tests/route_removal_during_failover_test/README.md

```diff
@@ -0,0 +1,55 @@
+# TE-13.2: gRIBI route DELETE during Failover 
+
+## Summary
+
+Validate gRIBI route flush during SSO
+
+## Procedure
+
+*   Connect ATE port-1 to DUT port-1, and ATE port-2 to DUT port-2.
+
+*   Create 64 L3 sub-interfaces under DUT port-2 and corresponding 64 L3 sub-interfaces on ATE port-2.
+
+*   Establish `gRIBI client` connection with DUT negotiating `FIB_ACK` as the requested ack_type.
+
+*   Install 64 L3 sub-interfaces IP to NextHopGroup(NHGID: `1`) containing all of the nexthops specified to ATE port-2.
+
+*   Inject `1000` IPv4Entries(IPBlock1: `198.18.196.1/22`) in default VRF with NHGID: `1`.
+
+*   Validate that the entries are installed as FIB_PROGRAMMED using getRPC.
+
+*   Send traffic from ATE port-1 to prefixes in IPBlock1 and ensure traffic flows 100% and reaches ATE port-2.
+
+*   Start flushing  IPv4Entries((IPBlock1: `198.18.196.1/22`) in default VRF with NHGID: `1`. Concurrently, trigger a supervisor switchover using gNOI `SwitchControlProcessor`  while IPBlock1 entries are only partially flushed.
+
+*   Following reconnection of the `gRIBI client` to a new master supervisor, validate if partially deleted entries of IPBlock1  are not present in the FIB using a get RPC.
+
+*   TODO: Check for coredumps in the DUT and validate that none are present post failover.
```

Comment:
Maybe we can use this gNOI method to check on if any new coredumps? 

https://github.com/openconfig/gnoi/blob/3df08ff738428a3c29db9052011a9b94c0be232b/file/file.proto#L129

Reply:
I think the end-state way of this working is that we would use `healthz`. Particularly:

* daemon crashes -- so `/components/component[name=GRIBIDAEMON]/healthz/state/status` is set to `UNHEALTHY`
* this creates a `gnoi.Healthz` event that has the associated artifacts, including logs and/or coredumps.
* a test could tell that the daemon went unhealthy (even if it recovers) by checking the `unhealthy-count` leaf.

This relies on healthz as the way to be able to achieve this, but is generalisable and discoverable after the crash.

The downside of this is that same dependency, `healthz` must be supported (which it might not be today).  `gnoi.File` seems a reasonable way to work around this - `Stat()` would be called with the argument of the path to the place that coredumps could be written.

There are a few things we have to think through here:

* if the files are on the secondary control-plane card (because we failed over) - what is the path? (does the primary mount the secondary's filesystem somewhere?)
* we need to know the path to the files - and need to be identify them as gRIBI cores.

As @xw-g pointed out in a quick chat on this, file is a much lower complexity dependency for the test though.

Reply:
@robshakir @xw-g I think we need to check for core dump for all of the tests. So, irrespective of the method for checking the core, I think this check need to be handled  by the FP/Ondatra test runner.

Reply:
updated with core file check.